bpf/lib/maps.h

```diff
@@ -165,6 +165,23 @@ struct bpf_elf_map __section_maps ENCRYPT_MAP = {
 	.max_elem	= 1,
 };
 
+#ifdef ENABLE_EGRESS_GATEWAY
+struct egress_key {
+	struct bpf_lpm_trie_key lpm_key;
+	__u32 sip;
+	__u32 dip;
+};
```

Comment:
Could you add `egress_info` and `egress_key` to `bpf/bpf_alignchecker.c`? That will allow the Cilium agent to check on startup that there is no discrepancy between the user and kernel definitions of the map.